### ISSUE TITLE
A Java Exception has occurred (Windows 10 x64)

### ISSUE THREAD

**Issue by Deweywsu** [paulpacifico/shutter-encoder#34]
The message in the title appears when the program is started.  I don't get anything past this error.  This has happened [before](https://github.com/paulpacifico/shutter-encoder/issues/22).  I tried to download the JRE file that appeared to solve that problem, but the download link isn't coming up.

I do not have Java installed on my machine.  When I ran the suggestion in issue #22, I got the exact same output:

"E:\Program Files\Shutter Encoder\JRE\bin\java.exe" -jar "E:\Program Files\Shutter Encoder\Shutter Encoder.exe"

Exception in thread "main" java.awt.AWTError: Assistive Technology not found: com.sun.java.accessibility.AccessBridge
        at java.desktop/java.awt.Toolkit.newAWTError(Unknown Source)
        at java.desktop/java.awt.Toolkit.fallbackToLoadClassForAT(Unknown Source)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
        at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
        at java.base/java.util.stream.ReferencePipeline.forEach(Unknown Source)
        at java.desktop/java.awt.Toolkit.loadAssistiveTechnologies(Unknown Source)
        at java.desktop/java.awt.Toolkit.getDefaultToolkit(Unknown Source)
        at java.desktop/java.awt.Window.getToolkit(Unknown Source)
        at java.desktop/java.awt.Window.init(Unknown Source)
        at java.desktop/java.awt.Window.<init>(Unknown Source)
        at java.desktop/java.awt.Frame.<init>(Unknown Source)
        at java.desktop/java.awt.Frame.<init>(Unknown Source)
        at java.desktop/javax.swing.JFrame.<init>(Unknown Source)
        at application.Shutter.<clinit>(Shutter.java:211)
Caused by: java.lang.ClassNotFoundException: com.sun.java.accessibility.AccessBridge
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
        at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Unknown Source)
        ... 19 more

Thanks for any help

**Comment by Deweywsu** [comment]
I was able to overcome this (for now) using the following process, which came from another software which exhibited the same issue:

1) Navigate to C:\Users\<username>
2) Open the file called .accessibility.properties using a standard text editor (Notepad is fine)
3)  Find the following lines in the config:
        assistive_technologies=com.sun.java.accessibility.AccessBridge
        screen_magnifier_present=true
4) Add a hashmark (#) in front of theses lines
5) Save your changes to the file, restart the Software

**Comment by paulpacifico** [comment]
Hi,
I'm sorry to be late, I am very busy.

It seems there is a sometimes a bug with installed Java environnement with the embedded one of Shutter.
I'm really grateful you found a solution I will keep this if someone get the same trouble.

Thanks,
Paul.

**Comment by weisborg** [comment]
Thanks for this fix Deweywsu!  This was driving me crazy.  I didn't even have any JAVA installed on my machine and was getting it.  Seeing those two lines to comment out with the # now I'm guessing it was a tool I installed over a month ago and uninstalled that our UI/UX person suggested I install to emulate a screen reader to test web pages for accessibility at my work.  I guess it didn't clean up after itself at all since there was another piece it left behind causing issues right after I uninstalled it too.

This really saved the day since I can't use Handbrake anymore.  I had upgraded to top of the line i7 Intel CPU and 64GB of RAM back at the start of COVID and it ended up being too fast peaking all 8 cores at 100% and throwing heat alarms.  I can tweak the affinity and such telling the process to only use every other core and try to limit the priority, but usually it can still only run 1-2 minutes before overheating the CPU.  Can't do any videos with it over 10-15 min long since they take too long to encode.  Shutter Encoded does the same thing with default settings, but if I force it to use the GPU to encode the CPU can hand it off and take a nap.
